index.html

```diff
@@ -232,6 +232,42 @@ <h3> <img src="./img/Rectangle 57.png" alt="points" width="10%"> skills</h3>
 
   
 </section>
+<!-- Contact form -->
+<section id="form-section">
+  <div class="form-container">
+    <div class="form-intro">
+      <div>
+        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      </div>
+    </div>
+    <div class="form-data">
+      <form action="https://formspree.io/f/xlekyljn" method="POST">
+          <input type="text" class="first-input" placeholder="First Name" maxlength="30" required>
+          <input type="text"  class="second-input"  placeholder="Last Name" maxlength="30" required> <br>
+          <input  type="email"  class="third-input"  id="input4"  placeholder="Email Address"  required> <br>
+          <textarea type="text" class="fourth-input"  maxlength="500" name="message"  placeholder="Write me something..." required ></textarea> <br>
+        <button type="submit" class="form-btn">Get in touch</button>
+      </form>
+    </div>
```

Comment:
- ***Your Design***
![image](https://github.com/Xghayor/Portfolio/assets/60083437/5d8eb9b5-ceba-4f51-bb72-66271036d094)

- [ ] Here I can see you have used two input fields for `name`, as `first name` and `last name`. But I can see in the given template that only the `Full Name` input field is used. Kindly fix this issue.

- ***Given Design***
![image](https://github.com/Xghayor/Portfolio/assets/60083437/4606b5e8-0836-40ad-b653-05bb06707e91)